app/views/users/index.html.erb

```diff
@@ -1,2 +1,18 @@
 
 <h1>Here the list of users </h1>
+
+<% @users.each do |user| %>
+  <div class="container">
+      <div class="row">
+          <div class="col-3 border my-3 p-3">
+              
+          </div>
+          <div class="col-9">
+              <div class ="d-flex justify-content-around align-items-baseline border m-3 p-3">
+                <h1><%= user.name %></h1>
+                <p>Number of posts : <%= user.posts_counter %></p>
```

Comment:
- [ ] I should be able to access the users show page by clicking the user's name.

Reply:
> * [ ]  I should be able to access the users show page by clicking the user's name.

please tell me where this in requirement?

Reply:
Please check my previous comment